docs/modules/ROOT/pages/explanation/synthesize.adoc

```diff
@@ -0,0 +1,122 @@
+= Synthesize a cluster
+
+include::partial$synthesize-macros.adoc[]
+
+This explanation goes into the details that are omitted in xref:how-tos/synthesize.adoc[synthesizing a cluster].
+
+== Creating a new cluster on control.vshn.net
+
+There's quite a few fields to fill out when creating a cluster.
+
+ID:: Choose a unique ID for the cluster.
+This field is limited to lower-case characters `a-z`, numbers `0-9` and `-`.
+The length is limited to 1-61 characters.
++
+NOTE: This field cannot be changed after creation, it's used as the Kubernetes object name for the cluster by Lieutenant.
++
+NOTE: We're planning to make the ID field optional, see ticket https://ticket.vshn.net/browse/PORTAL-49[PORTAL-49] in Jira.
```

Comment:
This is now already outdated as in the meantime PORTAL-49 has been implemented.

Reply:
Updated text accordingly